pkg/algorithm/nodeselect/v1alpha1/select_node.go

```diff
@@ -187,6 +210,96 @@ func (ac *Config) getBlockDevice() (*ndmapis.BlockDeviceList, error) {
 	if err != nil {
 		return nil, err
 	}
-	bdl := bdL.Filter(blockdevice.FilterClaimedDevices, blockdevice.FilterNonInactive)
+	bdl := bdL.Filter(blockdevice.FilterNonInactive)
```

Comment:
Better put following filter:
1. Get Non Active BDs which are claimed by the cstor-operator for this SPC.

Reply:
For very first time if we are processing the request is above kind of filter helpful?(none of the BD were in claimed)

Reply:
Filtering is happened in further steps